routes/Favourite.js

```diff
@@ -83,4 +83,32 @@ router.post('/isFavourite', urlencodedParser, async(req, res) => {
     }
 })
 
+router.get('/favouriteList', urlencodedParser, function(req, res) {
+    favourite.Favourite.find({},{
+        "_id" : 1,
+        "username" : 1,
+        "favourite" : 1
+    }).then(function(err, doc) {
+        if (err) { 
+            res.send(err); 
+        } else { 
+            res.send(doc); 
+        }
+    })
+});
+
+router.post('/listFavourite', urlencodedParser, async(req, res) => {
+    console.log(req.body.username);
```

Comment:
I have been going easy on `console.log` but the master code should only include minimal messages with well descriptive message.
one day you will need to go through the whole code base to clean up the `console.logs`